src/main/java/com/google/cloud/genomics/dataflow/readers/bam/ReaderOptions.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.genomics.dataflow.readers.bam;
+
+import htsjdk.samtools.ValidationStringency;
+
+import java.io.Serializable;
+
+/**
+ * A collection of options governing how BAM files are read.
+ */
+public class ReaderOptions implements Serializable{
+  ValidationStringency stringency = ValidationStringency.DEFAULT_STRINGENCY;
```

Comment:
I suggest removing the two set methods and making these final. That way the class is just data after construction.